src/Illuminate/Database/Migrations/MigrationCreator.php

```diff
@@ -70,12 +70,18 @@ public function create($name, $path, $table = null, $create = false)
      * Ensure that a migration with the given name doesn't already exist.
      *
      * @param  string  $name
+     * @param  string  $path
      * @return void
      *
      * @throws \InvalidArgumentException
      */
-    protected function ensureMigrationDoesntAlreadyExist($name)
+    protected function ensureMigrationDoesntAlreadyExist($name, $path)
```

Comment:
Changes to method signatures can break existing installations. Is this necessary, and if so, should this target the master branch?